examples/config/config

```diff
@@ -258,14 +260,20 @@ set history_disable_easter_egg 1
 @cbind  w            = event REQ_NEW_WINDOW
 
 # Page movement binds
-@cbind  j            = scroll vertical 20
-@cbind  k            = scroll vertical -20
-@cbind  h            = scroll horizontal -20
-@cbind  l            = scroll horizontal 20
+@cbind  j            = scroll vertical 100
+@cbind  J            = scroll vertical 50
```

Comment:
No, but the according VIM-actions make little sense in a browser, so I we might as well use them for this sort of "precise" scrolling.